model/src/main/resources/META-INF/openapi.yaml

```diff
@@ -90,6 +90,12 @@ components:
       value: "commit.committer=='nessie_committer'"
     expr_by_commitTime:
       value: "timestamp(commit.commitTime) > timestamp('2021-05-31T08:23:15Z')"
+    expr_by_commit_operations_in_namespace:
+      value: "operations.exists(op, op.key.startsWith('some.name.space.'))"
+    expr_by_commit_operations_table_name:
+      value: "operations.exists(op, op.name == 'BaseTable')"
```

Comment:
`op.name` looks a bit odd to me, it feels like the operation name... how about `op.entityName`?